<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Excel2TsWidget</class>
 <widget class="QWidget" name="Excel2TsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QPushButton" name="btnOpenOutputDir">
      <property name="geometry">
       <rect>
        <x>298</x>
        <y>18</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>输出目录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcelStatus">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>161</y>
        <width>32</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTs2Excel">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>161</y>
        <width>32</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSelectTs">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>146</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcel2Ts">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>132</y>
        <width>32</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSelectExcel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>146</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>E</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>283</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditExcelPath"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTsPath"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>279</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelFormat">
         <property name="text">
          <string>Excel文件格式选择：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox2Cols">
           <property name="text">
            <string>两列（源文|译文）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox3Cols">
           <property name="text">
            <string>三列（类名|源文|译文）</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
